<launch>

  <arg name="sim" default="false"/>
  
  <include file="$(find pmb2_lab_nav)/launch/pmb2_move.launch">
	<arg name="sim" value="$(arg sim)"/>
  </include>
  
  <include file="$(find pmb2_face)/launch/pmb2_talk.launch">
	<arg name="sim" value="$(arg sim)"/>  
  </include>
  
  <node pkg="pmb2_control" type="messages_node.py" name="messages_node" respawn="true" output="screen" />
  <node pkg="pmb2_control" type="check_node.py" name="check_node" respawn="true" output="screen" />
  
</launch>